auction/program/src/processor.rs

```diff
@@ -550,6 +595,8 @@ impl BidState {
         minimum: u64,
         instant_sale_price: Option<u64>,
         auction_state: &mut AuctionState,
+        decrease_rate: Option<u64>,
```

Comment:
@justinkook `auction_state`, `decrease_rate`, and `decrease_interval` are unused in this function. Any idea what the story is with them?